src/components/ViewFieldIngelogdExtraInfo/ViewFieldIngelogdExtraInfo.js

```diff
@@ -19,6 +14,15 @@ const eigenarenProperties = [
     'Opdrachtgever',
 ]
 
+/**
+ * Component that renders the ViewFieldIngelogdExtraInfo component and displays a owner list and a button to open the list in the admin area.
```

Comment:
I think 'to open the list' is a bit confusing. I would propose:
`Show extra information for authenticated users like the policy owners and a link to edit the policy object.`